data-node/src/test/java/org/graylog/datanode/integration/DatanodeSecuritySetupIT.java

```diff
@@ -86,6 +84,7 @@ void setUp() throws UnknownHostException {
             // configure transport security
             datanodeContainer.withEnv("GRAYLOG_DATANODE_TRANSPORT_CERTIFICATE", "datanode-transport-certificates.p12");
             datanodeContainer.withEnv("GRAYLOG_DATANODE_TRANSPORT_CERTIFICATE_PASSWORD", "password");
+            datanodeContainer.withEnv("GRAYLOG_DATANODE_INSECURE_STARTUP", "false");
```

Comment:
Is this flag needed here when default is false?

Reply:
Yes, because it is `true` in DatanodeContainerizedBackend.java.